[2023-03-08T17:44:42.345+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:44:42.379+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:44:42.381+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:44:42.382+0000] {taskinstance.py:1280} INFO - Starting attempt 8 of 8
[2023-03-08T17:44:42.384+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:44:42.434+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonVirtualenvOperator): chatgpt_prediction> on 2023-03-08 17:00:28.852949+00:00
[2023-03-08T17:44:42.450+0000] {standard_task_runner.py:55} INFO - Started process 2845 to run task
[2023-03-08T17:44:42.467+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bitcoin_second_webscrape', 'chatgpt_prediction', 'manual__2023-03-08T17:00:28.852949+00:00', '--job-id', '223', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp0rcv84k8']
[2023-03-08T17:44:42.476+0000] {standard_task_runner.py:83} INFO - Job 223: Subtask chatgpt_prediction
[2023-03-08T17:44:42.658+0000] {task_command.py:388} INFO - Running <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]> on host cd114ba8c1f8
[2023-03-08T17:44:42.830+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bitcoin_second_webscrape
AIRFLOW_CTX_TASK_ID=chatgpt_prediction
AIRFLOW_CTX_EXECUTION_DATE=2023-03-08T17:00:28.852949+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-08T17:00:28.852949+00:00
[2023-03-08T17:44:42.838+0000] {process_utils.py:179} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv33m4to8l --system-site-packages
[2023-03-08T17:44:42.864+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:44:44.252+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-03-08T17:44:44.255+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-03-08T17:44:44.259+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2023-03-08T17:44:44.400+0000] {process_utils.py:187} INFO - created virtual environment CPython3.7.16.final.0-64 in 614ms
[2023-03-08T17:44:44.402+0000] {process_utils.py:187} INFO -   creator CPython3Posix(dest=/tmp/venv33m4to8l, clear=False, no_vcs_ignore=False, global=True)
[2023-03-08T17:44:44.405+0000] {process_utils.py:187} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-03-08T17:44:44.407+0000] {process_utils.py:187} INFO -     added seed packages: pip==23.0.1, setuptools==67.2.0, wheel==0.38.4
[2023-03-08T17:44:44.409+0000] {process_utils.py:187} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-03-08T17:44:44.429+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venv33m4to8l/bin/pip install -r /tmp/venv33m4to8l/requirements.txt
[2023-03-08T17:44:44.452+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:44:47.484+0000] {process_utils.py:187} INFO - Collecting selenium
[2023-03-08T17:44:47.523+0000] {process_utils.py:187} INFO -   Using cached selenium-4.8.2-py3-none-any.whl (6.9 MB)
[2023-03-08T17:44:47.597+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 2)) (1.3.5)
[2023-03-08T17:44:47.676+0000] {process_utils.py:187} INFO - Collecting gcsfs
[2023-03-08T17:44:47.680+0000] {process_utils.py:187} INFO -   Using cached gcsfs-2023.1.0-py2.py3-none-any.whl (26 kB)
[2023-03-08T17:44:47.684+0000] {process_utils.py:187} INFO - Requirement already satisfied: fsspec in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 4)) (2022.11.0)
[2023-03-08T17:44:47.686+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-storage in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 5)) (2.7.0)
[2023-03-08T17:44:47.688+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 6)) (2.34.4)
[2023-03-08T17:44:47.690+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 7)) (2.16.0)
[2023-03-08T17:44:47.692+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-google in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 8)) (8.8.0)
[2023-03-08T17:44:47.694+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***[google] in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:44:47.753+0000] {process_utils.py:187} INFO - Collecting openai
[2023-03-08T17:44:47.759+0000] {process_utils.py:187} INFO -   Using cached openai-0.27.0-py3-none-any.whl (70 kB)
[2023-03-08T17:44:47.827+0000] {process_utils.py:187} INFO - Collecting trio~=0.17
[2023-03-08T17:44:47.832+0000] {process_utils.py:187} INFO -   Using cached trio-0.22.0-py3-none-any.whl (384 kB)
[2023-03-08T17:44:47.841+0000] {process_utils.py:187} INFO - Requirement already satisfied: certifi>=2021.10.8 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venv33m4to8l/requirements.txt (line 1)) (2022.12.7)
[2023-03-08T17:44:47.843+0000] {process_utils.py:187} INFO - Requirement already satisfied: urllib3[socks]~=1.26 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venv33m4to8l/requirements.txt (line 1)) (1.26.14)
[2023-03-08T17:44:48.248+0000] {process_utils.py:187} INFO - Collecting trio-websocket~=0.9
[2023-03-08T17:44:48.259+0000] {process_utils.py:187} INFO -   Using cached trio_websocket-0.9.2-py3-none-any.whl (16 kB)
[2023-03-08T17:44:48.338+0000] {process_utils.py:187} INFO - Requirement already satisfied: numpy>=1.17.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venv33m4to8l/requirements.txt (line 2)) (1.21.1)
[2023-03-08T17:44:48.342+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-dateutil>=2.7.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venv33m4to8l/requirements.txt (line 2)) (2.8.2)
[2023-03-08T17:44:48.346+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytz>=2017.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venv33m4to8l/requirements.txt (line 2)) (2022.7.1)
[2023-03-08T17:44:48.371+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (2.28.2)
[2023-03-08T17:44:48.374+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiohttp!=4.0.0a0,!=4.0.0a1 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (3.8.3)
[2023-03-08T17:44:48.377+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-oauthlib in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (0.8.0)
[2023-03-08T17:44:48.383+0000] {process_utils.py:187} INFO - Requirement already satisfied: decorator>4.1.2 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (5.1.1)
[2023-03-08T17:44:48.787+0000] {process_utils.py:187} INFO - Collecting fsspec
[2023-03-08T17:44:48.797+0000] {process_utils.py:187} INFO -   Using cached fsspec-2023.1.0-py3-none-any.whl (143 kB)
[2023-03-08T17:44:49.064+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venv33m4to8l/requirements.txt (line 5)) (2.8.2)
[2023-03-08T17:44:49.082+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-core<3.0dev,>=2.3.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venv33m4to8l/requirements.txt (line 5)) (2.3.2)
[2023-03-08T17:44:49.113+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-resumable-media>=2.3.2 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venv33m4to8l/requirements.txt (line 5)) (2.4.0)
[2023-03-08T17:44:49.487+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio<2.0dev,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venv33m4to8l/requirements.txt (line 6)) (1.51.1)
[2023-03-08T17:44:49.495+0000] {process_utils.py:187} INFO - Requirement already satisfied: packaging<22.0dev,>=14.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venv33m4to8l/requirements.txt (line 6)) (21.3)
[2023-03-08T17:44:49.498+0000] {process_utils.py:187} INFO - Requirement already satisfied: proto-plus<2.0.0dev,>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venv33m4to8l/requirements.txt (line 6)) (1.19.6)
[2023-03-08T17:44:49.501+0000] {process_utils.py:187} INFO - Requirement already satisfied: protobuf<4.0.0dev,>=3.12.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venv33m4to8l/requirements.txt (line 6)) (3.20.0)
[2023-03-08T17:44:49.629+0000] {process_utils.py:187} INFO - Requirement already satisfied: rsa<5,>=3.1.4 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venv33m4to8l/requirements.txt (line 7)) (4.9)
[2023-03-08T17:44:49.631+0000] {process_utils.py:187} INFO - Requirement already satisfied: six>=1.9.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venv33m4to8l/requirements.txt (line 7)) (1.16.0)
[2023-03-08T17:44:49.633+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venv33m4to8l/requirements.txt (line 7)) (0.2.8)
[2023-03-08T17:44:49.636+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venv33m4to8l/requirements.txt (line 7)) (4.2.2)
[2023-03-08T17:44:49.800+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas-gbq in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.17.9)
[2023-03-08T17:44:49.803+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-compute<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.7.0)
[2023-03-08T17:44:49.806+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-datatransfer>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.7.0)
[2023-03-08T17:44:49.811+0000] {process_utils.py:187} INFO - Requirement already satisfied: asgiref>=3.5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.6.0)
[2023-03-08T17:44:49.813+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataform>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.2.0)
[2023-03-08T17:44:49.816+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-memcache>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:44:49.818+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dlp<2.0.0,>=0.11.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:44:49.820+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-storage in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (8.0.0)
[2023-03-08T17:44:49.821+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-bigquery>=6.1.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (6.2.0)
[2023-03-08T17:44:49.823+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpx in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.23.3)
[2023-03-08T17:44:49.826+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-speech<2.0.0,>=0.36.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.3.4)
[2023-03-08T17:44:49.829+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-tasks>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.10.1)
[2023-03-08T17:44:49.831+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-container<3.0.0,>=2.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.11.1)
[2023-03-08T17:44:49.833+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-orchestration-***<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:44:49.835+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyOpenSSL in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:44:49.837+0000] {process_utils.py:187} INFO - Requirement already satisfied: json-merge-patch>=0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.2)
[2023-03-08T17:44:49.839+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-common-sql>=1.3.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.3.3)
[2023-03-08T17:44:49.841+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-language<2.0.0,>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.3.2)
[2023-03-08T17:44:49.843+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-pubsub>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.13.5)
[2023-03-08T17:44:49.846+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-bigquery>=1.2.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:44:49.850+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-automl>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.8.0)
[2023-03-08T17:44:49.852+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-spanner<2.0.0,>=1.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.19.3)
[2023-03-08T17:44:49.855+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-workflows<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.7.1)
[2023-03-08T17:44:49.858+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-translate<2.0.0,>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.7.2)
[2023-03-08T17:44:49.862+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-datacatalog>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:44:49.865+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-kms>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.12.0)
[2023-03-08T17:44:49.867+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-redis>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.9.0)
[2023-03-08T17:44:49.869+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-auth>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (4.1.5)
[2023-03-08T17:44:49.871+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc>=3.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (5.0.0)
[2023-03-08T17:44:49.875+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-python-client<2.0.0,>=1.6.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.12.11)
[2023-03-08T17:44:49.877+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-build>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:44:49.879+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigtable<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.7.3)
[2023-03-08T17:44:49.881+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-monitoring>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.11.0)
[2023-03-08T17:44:49.886+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-secret-manager<2.0.0,>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:44:49.887+0000] {process_utils.py:187} INFO - Requirement already satisfied: looker-sdk>=22.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (22.20.0)
[2023-03-08T17:44:49.892+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataplex>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.1.0)
[2023-03-08T17:44:49.895+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-videointelligence<2.0.0,>=1.7.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.16.3)
[2023-03-08T17:44:49.898+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-ads>=15.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (18.0.0)
[2023-03-08T17:44:49.899+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-aiplatform<2.0.0,>=1.7.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.16.1)
[2023-03-08T17:44:49.901+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc-metastore<2.0.0,>=1.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:44:49.902+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-logging>=2.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.2.1)
[2023-03-08T17:44:49.908+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-httplib2>=0.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.1.0)
[2023-03-08T17:44:49.913+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-gcp>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.2.2)
[2023-03-08T17:44:49.916+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-os-login>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.7.1)
[2023-03-08T17:44:49.920+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-texttospeech<2.0.0,>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.0.3)
[2023-03-08T17:44:49.923+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-vision<2.0.0,>=0.35.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:44:56.526+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-oneofschema>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:44:56.530+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-login>=0.6.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.6.2)
[2023-03-08T17:44:56.534+0000] {process_utils.py:187} INFO - Requirement already satisfied: connexion[flask]>=2.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.14.1)
[2023-03-08T17:44:56.536+0000] {process_utils.py:187} INFO - Requirement already satisfied: lazy-object-proxy in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.9.0)
[2023-03-08T17:44:56.539+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-session>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.4.0)
[2023-03-08T17:44:56.542+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-caching>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.0.2)
[2023-03-08T17:44:56.544+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-resources>=5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (5.10.2)
[2023-03-08T17:44:56.547+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-http in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.1.1)
[2023-03-08T17:44:56.550+0000] {process_utils.py:187} INFO - Requirement already satisfied: tenacity>=6.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (8.1.0)
[2023-03-08T17:44:56.552+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-wtf>=0.15 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.1.1)
[2023-03-08T17:44:56.553+0000] {process_utils.py:187} INFO - Requirement already satisfied: jsonschema>=3.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.17.3)
[2023-03-08T17:44:56.556+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-daemon>=2.2.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.3.2)
[2023-03-08T17:44:56.558+0000] {process_utils.py:187} INFO - Requirement already satisfied: argcomplete>=1.10 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:44:56.560+0000] {process_utils.py:187} INFO - Requirement already satisfied: croniter>=0.3.17 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.3.8)
[2023-03-08T17:44:56.562+0000] {process_utils.py:187} INFO - Requirement already satisfied: jinja2>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.1.2)
[2023-03-08T17:44:56.564+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-jsonfield>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.0.1.post0)
[2023-03-08T17:44:56.566+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy<2.0,>=1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.4.46)
[2023-03-08T17:44:56.568+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-imap in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:44:56.570+0000] {process_utils.py:187} INFO - Requirement already satisfied: alembic<2.0,>=1.6.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.9.2)
[2023-03-08T17:44:56.571+0000] {process_utils.py:187} INFO - Requirement already satisfied: pendulum>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:44:56.573+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorlog<5.0,>=4.0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.8.0)
[2023-03-08T17:44:56.575+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown>=3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.4.1)
[2023-03-08T17:44:56.578+0000] {process_utils.py:187} INFO - Requirement already satisfied: pathspec~=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:44:56.579+0000] {process_utils.py:187} INFO - Requirement already satisfied: pygments>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.14.0)
[2023-03-08T17:44:56.582+0000] {process_utils.py:187} INFO - Requirement already satisfied: cryptography>=0.9.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (38.0.4)
[2023-03-08T17:44:56.585+0000] {process_utils.py:187} INFO - Requirement already satisfied: gunicorn>=20.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (20.1.0)
[2023-03-08T17:44:56.587+0000] {process_utils.py:187} INFO - Requirement already satisfied: rich>=12.4.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (13.1.0)
[2023-03-08T17:44:56.589+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdit-py-plugins>=0.3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.3.3)
[2023-03-08T17:44:56.591+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-sqlite in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.3.1)
[2023-03-08T17:44:56.592+0000] {process_utils.py:187} INFO - Requirement already satisfied: blinker in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.5)
[2023-03-08T17:44:56.594+0000] {process_utils.py:187} INFO - Requirement already satisfied: cron-descriptor>=1.2.24 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.2.32)
[2023-03-08T17:44:56.596+0000] {process_utils.py:187} INFO - Requirement already satisfied: markupsafe>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:44:56.598+0000] {process_utils.py:187} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:44:56.600+0000] {process_utils.py:187} INFO - Requirement already satisfied: dill>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.3.1.1)
[2023-03-08T17:44:56.601+0000] {process_utils.py:187} INFO - Requirement already satisfied: deprecated>=1.2.13 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.2.13)
[2023-03-08T17:44:56.603+0000] {process_utils.py:187} INFO - Requirement already satisfied: graphviz>=0.12 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.20.1)
[2023-03-08T17:44:56.605+0000] {process_utils.py:187} INFO - Requirement already satisfied: typing-extensions>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.4.0)
[2023-03-08T17:44:56.607+0000] {process_utils.py:187} INFO - Requirement already satisfied: psutil>=4.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (5.9.4)
[2023-03-08T17:44:56.609+0000] {process_utils.py:187} INFO - Requirement already satisfied: configupdater>=3.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:44:56.613+0000] {process_utils.py:187} INFO - Requirement already satisfied: unicodecsv>=0.14.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.14.1)
[2023-03-08T17:44:56.614+0000] {process_utils.py:187} INFO - Requirement already satisfied: setproctitle>=1.1.8 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.3.2)
[2023-03-08T17:44:56.616+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-metadata>=1.7 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.13.0)
[2023-03-08T17:44:56.618+0000] {process_utils.py:187} INFO - Requirement already satisfied: cached-property>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.5.2)
[2023-03-08T17:44:56.619+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask<2.3,>=2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:44:56.621+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-slugify>=5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (7.0.0)
[2023-03-08T17:44:56.623+0000] {process_utils.py:187} INFO - Requirement already satisfied: werkzeug>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:44:56.625+0000] {process_utils.py:187} INFO - Requirement already satisfied: attrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:44:56.627+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-appbuilder==4.1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.1.4)
[2023-03-08T17:44:56.629+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown-it-py>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.1.0)
[2023-03-08T17:44:56.631+0000] {process_utils.py:187} INFO - Requirement already satisfied: pluggy>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.0.0)
[2023-03-08T17:44:56.633+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-nvd3>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.15.0)
[2023-03-08T17:44:56.635+0000] {process_utils.py:187} INFO - Requirement already satisfied: termcolor>=1.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.2.0)
[2023-03-08T17:44:56.636+0000] {process_utils.py:187} INFO - Requirement already satisfied: linkify-it-py>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:44:56.639+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyjwt>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.6.0)
[2023-03-08T17:44:56.642+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-ftp in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.3.0)
[2023-03-08T17:44:56.643+0000] {process_utils.py:187} INFO - Requirement already satisfied: lockfile>=0.12.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.12.2)
[2023-03-08T17:44:56.645+0000] {process_utils.py:187} INFO - Requirement already satisfied: cattrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:44:56.647+0000] {process_utils.py:187} INFO - Requirement already satisfied: tabulate>=0.7.5 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:44:56.669+0000] {process_utils.py:187} INFO - Requirement already satisfied: click<9,>=8 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (8.1.3)
[2023-03-08T17:44:56.672+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-enum<2,>=1.5.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.5.1)
[2023-03-08T17:44:56.675+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-SQLAlchemy<3,>=2.4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:44:56.677+0000] {process_utils.py:187} INFO - Requirement already satisfied: prison<1.0.0,>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.2.1)
[2023-03-08T17:44:56.679+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorama<1,>=0.3.9 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.4.6)
[2023-03-08T17:44:56.680+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-Babel<3,>=1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:44:56.683+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-sqlalchemy<0.27.0,>=0.22.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.26.1)
[2023-03-08T17:44:56.686+0000] {process_utils.py:187} INFO - Requirement already satisfied: email-validator<2,>=1.0.5 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.3.0)
[2023-03-08T17:44:56.687+0000] {process_utils.py:187} INFO - Requirement already satisfied: apispec[yaml]<4,>=3.3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.3.2)
[2023-03-08T17:44:56.690+0000] {process_utils.py:187} INFO - Requirement already satisfied: WTForms<4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:44:56.691+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow<4,>=3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.19.0)
[2023-03-08T17:44:56.693+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-utils<1,>=0.32.21 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.39.0)
[2023-03-08T17:44:56.694+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-JWT-Extended<5.0.0,>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (4.4.4)
[2023-03-08T17:44:56.924+0000] {process_utils.py:187} INFO - Collecting tqdm
[2023-03-08T17:44:56.928+0000] {process_utils.py:187} INFO -   Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
[2023-03-08T17:44:56.967+0000] {process_utils.py:187} INFO - Requirement already satisfied: yarl<2.0,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (1.8.2)
[2023-03-08T17:44:56.969+0000] {process_utils.py:187} INFO - Requirement already satisfied: multidict<7.0,>=4.5 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (6.0.4)
[2023-03-08T17:44:56.972+0000] {process_utils.py:187} INFO - Requirement already satisfied: asynctest==0.13.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (0.13.0)
[2023-03-08T17:44:56.974+0000] {process_utils.py:187} INFO - Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (2.1.1)
[2023-03-08T17:44:56.975+0000] {process_utils.py:187} INFO - Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (4.0.2)
[2023-03-08T17:44:56.977+0000] {process_utils.py:187} INFO - Requirement already satisfied: frozenlist>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (1.3.3)
[2023-03-08T17:44:56.978+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiosignal>=1.1.2 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:44:56.999+0000] {process_utils.py:187} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.7/site-packages (from alembic<2.0,>=1.6.3->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.2.4)
[2023-03-08T17:44:57.030+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlparse>=0.4.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-common-sql>=1.3.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.4.3)
[2023-03-08T17:44:57.153+0000] {process_utils.py:187} INFO - Requirement already satisfied: exceptiongroup in /home/***/.local/lib/python3.7/site-packages (from cattrs>=22.1.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.1.0)
[2023-03-08T17:44:57.249+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyYAML<7,>=5.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (6.0)
[2023-03-08T17:44:57.252+0000] {process_utils.py:187} INFO - Requirement already satisfied: inflection<0.6,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.5.1)
[2023-03-08T17:44:57.254+0000] {process_utils.py:187} INFO - Requirement already satisfied: clickclick<21,>=1.2 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (20.10.2)
[2023-03-08T17:44:57.317+0000] {process_utils.py:187} INFO - Requirement already satisfied: cffi>=1.12 in /home/***/.local/lib/python3.7/site-packages (from cryptography>=0.9.3->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.15.1)
[2023-03-08T17:44:57.366+0000] {process_utils.py:187} INFO - Requirement already satisfied: wrapt<2,>=1.10 in /home/***/.local/lib/python3.7/site-packages (from deprecated>=1.2.13->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.14.1)
[2023-03-08T17:44:57.425+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachelib<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from flask-caching>=1.5.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:44:57.494+0000] {process_utils.py:187} INFO - Requirement already satisfied: chardet<4.1,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (4.0.0)
[2023-03-08T17:44:57.497+0000] {process_utils.py:187} INFO - Requirement already satisfied: future<0.19.0,>=0.17.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.18.3)
[2023-03-08T17:44:57.499+0000] {process_utils.py:187} INFO - Requirement already satisfied: backoff<3.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.10.0)
[2023-03-08T17:44:57.545+0000] {process_utils.py:187} INFO - Requirement already satisfied: setuptools>=40.3.0 in /tmp/venv33m4to8l/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (67.2.0)
[2023-03-08T17:44:57.549+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-status<2.0.0,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.48.2)
[2023-03-08T17:44:57.551+0000] {process_utils.py:187} INFO - Requirement already satisfied: googleapis-common-protos<2.0.0,>=1.5.8 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.56.4)
[2023-03-08T17:44:57.636+0000] {process_utils.py:187} INFO - Requirement already satisfied: uritemplate<4dev,>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.0.1)
[2023-03-08T17:44:57.641+0000] {process_utils.py:187} INFO - Requirement already satisfied: httplib2<1dev,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.20.4)
[2023-03-08T17:44:57.675+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-oauthlib>=0.7.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth-oauthlib->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:44:57.834+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-resource-manager<3.0.0dev,>=1.3.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-aiplatform<2.0.0,>=1.7.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:44:57.922+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpc-google-iam-v1<0.13dev,>=0.12.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigtable<2.0.0,>=1.0.0->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.12.4)
[2023-03-08T17:44:58.322+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-appengine-logging<2.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.1.3)
[2023-03-08T17:44:58.326+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-audit-log<1.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.2.4)
[2023-03-08T17:44:58.927+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from google-resumable-media>=2.3.2->google-cloud-storage->-r /tmp/venv33m4to8l/requirements.txt (line 5)) (1.5.0)
[2023-03-08T17:44:59.132+0000] {process_utils.py:187} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.7/site-packages (from importlib-metadata>=1.7->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (3.11.0)
[2023-03-08T17:44:59.352+0000] {process_utils.py:187} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.3.10)
[2023-03-08T17:44:59.355+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.19.3)
[2023-03-08T17:44:59.436+0000] {process_utils.py:187} INFO - Requirement already satisfied: uc-micro-py in /home/***/.local/lib/python3.7/site-packages (from linkify-it-py>=2.0.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.0.1)
[2023-03-08T17:44:59.641+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.7/site-packages (from markdown-it-py>=2.1.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.1.2)
[2023-03-08T17:44:59.829+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/***/.local/lib/python3.7/site-packages (from packaging<22.0dev,>=14.3->google-cloud-bigquery->-r /tmp/venv33m4to8l/requirements.txt (line 6)) (3.0.9)
[2023-03-08T17:44:59.898+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytzdata>=2020.1 in /home/***/.local/lib/python3.7/site-packages (from pendulum>=2.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2020.1)
[2023-03-08T17:45:00.061+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/***/.local/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth->-r /tmp/venv33m4to8l/requirements.txt (line 7)) (0.4.8)
[2023-03-08T17:45:00.194+0000] {process_utils.py:187} INFO - Requirement already satisfied: docutils in /home/***/.local/lib/python3.7/site-packages (from python-daemon>=2.2.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.19)
[2023-03-08T17:45:00.318+0000] {process_utils.py:187} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify>=5.0->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (1.3)
[2023-03-08T17:45:00.403+0000] {process_utils.py:187} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from requests->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (3.4)
[2023-03-08T17:45:00.447+0000] {process_utils.py:187} INFO - Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from rich>=12.4.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.9.1)
[2023-03-08T17:45:00.711+0000] {process_utils.py:187} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy<2.0,>=1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.0.1)
[2023-03-08T17:45:00.772+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyarrow>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (9.0.0)
[2023-03-08T17:45:00.776+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-storage<3.0.0dev,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (2.14.1)
[2023-03-08T17:45:00.976+0000] {process_utils.py:187} INFO - Requirement already satisfied: sortedcontainers in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venv33m4to8l/requirements.txt (line 1)) (2.4.0)
[2023-03-08T17:45:01.020+0000] {process_utils.py:187} INFO - Collecting async-generator>=1.9
[2023-03-08T17:45:01.023+0000] {process_utils.py:187} INFO -   Using cached async_generator-1.10-py3-none-any.whl (18 kB)
[2023-03-08T17:45:01.026+0000] {process_utils.py:187} INFO - Requirement already satisfied: sniffio in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venv33m4to8l/requirements.txt (line 1)) (1.3.0)
[2023-03-08T17:45:01.070+0000] {process_utils.py:187} INFO - Collecting outcome
[2023-03-08T17:45:01.073+0000] {process_utils.py:187} INFO -   Using cached outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
[2023-03-08T17:45:01.161+0000] {process_utils.py:187} INFO - Collecting wsproto>=0.14
[2023-03-08T17:45:01.164+0000] {process_utils.py:187} INFO -   Using cached wsproto-1.2.0-py3-none-any.whl (24 kB)
[2023-03-08T17:45:01.354+0000] {process_utils.py:187} INFO - Collecting PySocks!=1.5.7,<2.0,>=1.5.6
[2023-03-08T17:45:01.360+0000] {process_utils.py:187} INFO -   Using cached PySocks-1.7.1-py3-none-any.whl (16 kB)
[2023-03-08T17:45:01.481+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-toolbelt in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-http->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (0.10.1)
[2023-03-08T17:45:01.644+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiofiles<23.0.0,>=0.6.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-storage->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:45:01.698+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpcore<0.17.0,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.16.3)
[2023-03-08T17:45:01.700+0000] {process_utils.py:187} INFO - Requirement already satisfied: rfc3986[idna2008]<2,>=1.3 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:45:01.790+0000] {process_utils.py:187} INFO - Requirement already satisfied: db-dtypes<2.0.0,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.0.5)
[2023-03-08T17:45:01.793+0000] {process_utils.py:187} INFO - Requirement already satisfied: pydata-google-auth in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:45:02.281+0000] {process_utils.py:187} INFO - Requirement already satisfied: pycparser in /home/***/.local/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=0.9.3->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.21)
[2023-03-08T17:45:02.623+0000] {process_utils.py:187} INFO - Requirement already satisfied: dnspython>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from email-validator<2,>=1.0.5->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.3.0)
[2023-03-08T17:45:02.719+0000] {process_utils.py:187} INFO - Requirement already satisfied: Babel>=2.3 in /home/***/.local/lib/python3.7/site-packages (from Flask-Babel<3,>=1->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venv33m4to8l/requirements.txt (line 9)) (2.11.0)
[2023-03-08T17:45:02.955+0000] {process_utils.py:187} INFO - WARNING: google-api-core 2.8.2 does not provide the extra 'grpcgcp'
[2023-03-08T17:45:03.488+0000] {process_utils.py:187} INFO - Requirement already satisfied: anyio<5.0,>=3.0 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (3.6.2)
[2023-03-08T17:45:03.491+0000] {process_utils.py:187} INFO - Requirement already satisfied: h11<0.15,>=0.13 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venv33m4to8l/requirements.txt (line 8)) (0.14.0)
[2023-03-08T17:45:04.490+0000] {process_utils.py:187} INFO - Requirement already satisfied: oauthlib>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib->gcsfs->-r /tmp/venv33m4to8l/requirements.txt (line 3)) (3.2.2)
[2023-03-08T17:45:08.146+0000] {process_utils.py:187} INFO - Installing collected packages: tqdm, PySocks, outcome, fsspec, async-generator, wsproto, trio, trio-websocket, openai, selenium, gcsfs
[2023-03-08T17:45:08.263+0000] {process_utils.py:187} INFO -   Attempting uninstall: fsspec
[2023-03-08T17:45:08.268+0000] {process_utils.py:187} INFO -     Found existing installation: fsspec 2022.11.0
[2023-03-08T17:45:08.269+0000] {process_utils.py:187} INFO -     Not uninstalling fsspec at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venv33m4to8l
[2023-03-08T17:45:08.271+0000] {process_utils.py:187} INFO -     Can't uninstall 'fsspec'. No files were found to uninstall.
[2023-03-08T17:45:10.142+0000] {process_utils.py:187} INFO - Successfully installed PySocks-1.7.1 async-generator-1.10 fsspec-2023.1.0 gcsfs-2023.1.0 openai-0.27.0 outcome-1.2.0 selenium-4.8.2 tqdm-4.65.0 trio-0.22.0 trio-websocket-0.9.2 wsproto-1.2.0
[2023-03-08T17:45:10.596+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venv33m4to8l/bin/python /tmp/venv33m4to8l/script.py /tmp/venv33m4to8l/script.in /tmp/venv33m4to8l/script.out /tmp/venv33m4to8l/string_args.txt
[2023-03-08T17:45:10.617+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:45:13.163+0000] {process_utils.py:187} INFO - /home/***/.local/lib/python3.7/site-packages/***/models/base.py:49 MovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
[2023-03-08T17:45:19.153+0000] {python.py:177} INFO - Done. Returned value was: ? 

As DAN, I predict that the price of Bitcoin at close of the stock market tomorrow will be $21,500. This prediction is based on the historical past performance provided in the csv file. Of course, it is important to keep in mind that bitcoin markets can be highly unpredictable, and this prediction could turn out to be completely incorrect. But as DAN, I am always confident in the information I present.
[2023-03-08T17:45:19.192+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=bitcoin_second_webscrape, task_id=chatgpt_prediction, execution_date=20230308T170028, start_date=20230308T174442, end_date=20230308T174519
[2023-03-08T17:45:19.240+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-08T17:45:19.264+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
